src/molecules/pagination/pagination.tsx

```diff
@@ -146,6 +153,20 @@ const Pagination: React.FC<PaginationProps> = (props) => {
       >
         <Icon icon="ChevronRight" />
       </PaginationLink>
+      {
+        total >= 3
+          ? (
+            <PaginationLink
+              data-testid="last"
+              onClick={(): void => (!isLastPage ? onChange(paginate.endPage) : undefined)}
```

Comment:
`: void` shouldn't be needed, types can be derived here. Can you remove those (and from the old PaginationLink)?

Reply:
fixed